api/api.go

```diff
@@ -1184,6 +1184,9 @@ func parseQueryMeta(resp *http.Response, q *QueryMeta) error {
 	if err != nil {
 		return fmt.Errorf("Failed to parse X-Nomad-LastContact: %v", err)
 	}
+	if last > uint64(math.MaxInt64) {
```

Comment:
math.MaxInt64 is an untyped constant, do we need a cast here?

Reply:
> I think it's because this check is to see if last (which is a uint64) is greater than time.Duration (which is int64)
https://github.com/hashicorp/nomad-enterprise/pull/1433#discussion_r1583156618

Because we do `q.LastContact = time.Duration(last) * time.Millisecond` on the next line.

Reply:
Right, but I think @shoenig's point is that the `math.MaxInt64` doesn't need the cast here. It compiles without it.

Fixed.